docs/topics/con_key-vertx-concepts.adoc

```diff
@@ -11,14 +11,14 @@ A reactive system, as defined in the link:http://reactivemanifesto.org[reactive
 
 Elastic:: The system remains responsive under varying workload, with individual components scaled and load-balanced as necessary to accommodate the differences in workload. Elastic application deliver the same quality of service regardless of the number of requests they receive at the same time.
 
-Resilient:: The system remains responsive in case any of its individual components fails. Components are isolated from each other, with each individual component being able to quickly recover after failure occurs. Failure of a single component should never prevent other components from functioning beyond a reasonable amount of time. This prevents cascading failure, where the failure of an isolated components causes other components to become blocked and gradually fail.
+Resilient:: The system remains responsive in case any of its individual components fails. Components are isolated from each other, with each individual component being able to quickly recover after failure occurs. Failure of a single component should never prevent other components from functioning in a reasonable amount of time. This prevents cascading failure, where the failure of an isolated components causes other components to become blocked and gradually fail.
```

Comment:
I think this is a change in the meaning, but not sure what the original meaning was :-) I think it was supposed to say that failure of single component won't prevent other components from functioning, except of a short period of time around the moment of failure. Agree the original usage of "beyond" is a poor stylistic choice.

Reply:
I arrived at the same conclusion as you did after reading the context around it. Now that you put it like this, I think I'll rewrite the sentence completely.